recipes/uriparser/all/conanfile.py

```diff
@@ -35,6 +37,11 @@ def _source_subfolder(self):
     def _build_subfolder(self):
         return "build_subfolder"
 
+    def export_sources(self):
```

Comment:
Is there a template or specialized `ConanFile` feature in Conan 🤔 Maybe with python requires?

We have a lot of methods whicch are almost always the same in CCI, I would be really convenient to do

```py
class UriparserConan(C3iCMakeConanFile):
    pass
```

And just have everything?
- folder properties
- export_sources
- source
- config_option??? (it's almost always the same)

 Not a python expert but C++ override/virtual feel for the methods